frontend_new/src/components/Res.js

```diff
@@ -19,19 +21,25 @@ function Res(props) {
     color: "#9f9f9f",
   };
 
+  const [time_delay, setDelay]= useState(1);
+
   return (
     <List>
       {!resStatus &&
         res.result.result.map(({ _id, _score, _source }) => (
           <React.Fragment key={_id}>
-            <ListItem button>
-              <ListItemText
-                secondaryTypographyProps={{ style: textColor2 }}
-                primaryTypographyProps={{ style: textColor }}
-                primary={_source.title}
-                secondary={_source.summary}
+            <Grow in={true} timeout={time_delay}>
+
+            
+            <ListItem button onClick={() => console.log(_source.link)}>
+            {() => setDelay(time_delay+200)}
```

Comment:
**[Required, Maintainability, Magic Values]** Ideally, we should create a constant that describes exact what this value of `200` actually means. Having it inline could mean anything and it becomes a maintainability issue.